### DISCUSSION TITLE
Heat of formation

### DISCUSSION THREAD

**Discussion by lihy213** [firemodels/fds#13873] [Q&A]
Hello team,
In my knowlege, the heat of formation is defined as the producted or released heat when the specie formed from a single specie. So the $\Delta H_F$ should be zero for the single specie. But in FDS, for example, the $\Delta H_F$ of Na is 107.5. I'm confused about it. Samely, I also found that value in some references.

**Comment by drjfloyd** [comment]
The heat of formation is zero when the species is formed at its reference state. For Na that state is solid not gas.

**Reply by lihy213** [reply]
Thanks!I understand! Zero for a single species means the state of that species stable rather the sodium gas is not always stable. Is it right?

**Reply by drjfloyd** [reply]
At standard temperature and pressure sodium is a solid and not a gas; therefore, the heat of formation is 0 for the solid. For oxygen, nitrogen, helium, etc. They are a gas at standard temperature and pressure so they have a heat of formation that is 0 for the gas.
